docs/non-normative/how-to-write-conventions/resource-and-entities.md

```diff
@@ -0,0 +1,301 @@
+# Resource and Entities
+
+Table of Contents
+
+- [Modelling Guide](#modelling-guide)
+- FAQ
+  - [When to define a new entity](#when-to-define-a-new-entity)
+  - [When to define a new "is-a" relationship](#when-to-define-an-is-a-relationship-vs-extending-descriptive-attributes)
+  - [How to define identifying attributes](#how-to-define-identifying-attributes)
+  - [How to namespace entities](#how-to-namespace-entities)
+- [Background](#background-resource-and-entities)
+
+## Modelling Guide
+
+To define an entity, create a new semantic convention model file, with a group type as `entity`, for example:
+
+`model/{my_domain}/entities.yaml`:
+
+```yaml
+groups:
+  - id: entity.my_entity
+    type: entity
+    stability: development
+    name: my_entity
+    brief: >
+      A description of my_entity here.
+    attributes:
+      - ref: some.attribute
+        role: identifying
+      - ref: some.other_attribute
+        role: descriptive
+        ...
+```
+
+Here, the attributes field contains all attributes of the Entity.
+The `role` of each attribute determines if it is identifying or
+descriptive. See
+[How to define identifying attributes?](#how-to-define-identifying-attributes)
+for details on what these mean.
+
+> Note: Declaring Entity relationships is not yet supported.
```

Comment:
The document later talks a lot about "is-a" relationship, so this note is confusing. If declaring relationships is not supported then what exactly is supported for relationships? Is it used purely as an explanation mechanism in this document?

Reply:
Yes.  This is to help folks understand better what a Resource with `container` and `k8s.container` would mean, for example, or another example why a resource might have `k8s.node`, `aws.ecs` and `host` entities.

Another way to phrase this - I think understanding is-a relationship, at a high level, is needed to model entities today.  We can flesh out exactly how to express that distinction in Phase 2.